internal/operation/main.go

```diff
@@ -35,7 +34,7 @@ func AssignBodyAttributes(o Operation, attributes interface{}) error {
 	flags := o.GetFlags()
 
 	for _, v := range *flags.Schema {
-		if v.Name == "id" {
+		if v.Name == "id" || v.Name == "id_or_slug" {
```

Comment:
This has been solved in a more elegant way in another PR I'm working on. For now, I just decided to keep like this to fix a bug on the value assignment for the project id/slug.